deploy/cron.js

```diff
@@ -98,22 +114,24 @@ var calculateNextDailyRunTimeOffset = function(){
 }
 
 logger.info("starting cron.js!");
-// api_run();
-// daily_run();
-// hourly_run();
-// realtime_run();
-// //api
-// setInterval(api_run,1000 * 60 * 60 * 24)
-// //daily
-// setTimeout(() => {
-// 	// Run at 10 AM UTC, then every 24 hours afterwards
-// 	// daily_run();
-// 	// setInterval(daily_run, 1000 * 60 * 60 * 24);
-// 	//api
-// 	api_run();
-// 	setInterval(api_run,1000 * 60 * 60 * 24)
-// }, calculateNextDailyRunTimeOffset());
-// //hourly
-// setInterval(hourly_run,1000 * 60 * 60);
-// //realtime
-// setInterval(realtime_run,1000 * 60 * 5);
+api_run();
+api_ua_run();
+daily_run();
+hourly_run();
+realtime_run();
+// daily
+setTimeout(() => {
```

Comment:
Bringing in `api_run` and`api_ua_run` so that they run at the same interval as daily. Otherwise we are dependent on the time that the app was staged in cloud.